iree/tools/run_mlir_main.cc

```diff
@@ -258,6 +265,21 @@ StatusOr<std::string> PrepareModule(
   return binary_contents;
 }
 
+// Returns a splitted input values from `filename` using comma as separater.
+StatusOr<std::vector<std::string>> GetInputValues(const std::string& filename) {
+  std::string error_message;
+  auto file = mlir::openInputFile(filename, &error_message);
+  if (!file) {
+    return NotFoundErrorBuilder(IREE_LOC)
+           << "Unable to open input file " << filename << ": " << error_message;
```

Comment:
Nit: put filename in quotes

```suggestion
           << "Unable to open input file '" << filename << "': " << error_message;
```

Reply:
Done